### ISSUE TITLE
Add a feed of edits

### ISSUE THREAD

**Issue by hughbris** [hughbris/grav-plugin-pushy#33]
Purely sprung from self-interest, provide RSS/Atom feed(s) of changes made to the site through Pushy. This suits me because it allows me to keep a caring eye on client editing activity, and to access this via a feedreader app on my phone device (which I'm enjoying again, over all the other channels and sources on offer).

I guess providing security options is the big architectural question with this feature.

**Comment by pamtbaau** [comment]
Why not add an extra tab in Admin in the Pushy section that lists a `git log` response? That might also be useful for the users of Pushy.

If you want more then a list of commit messages, it will be a bit more work.

I don't think there is any added security issue this way.

**Comment by hughbris** [comment]
> If you want more then a list of commit messages, it will be a bit more work.

Inline details would be nice but commit messages and a link will suffice.

In any case, it occurred to me Github (and maybe other platforms) might [expose feeds for public repositories](https://stackoverflow.com/questions/7353538/setting-up-a-github-commit-rss-feed) (they do), as well as private repositories (apparently but I can't get the suggested token authentication to work, still hoping).

* [Pushy commits feed](https://github.com/hughbris/grav-plugin-pushy/commits.atom)
* [Pushy develop branch commits feed](https://github.com/hughbris/grav-plugin-pushy/commits/develop.atom)

Adding `?token=ghp_PAT` doesn't work as described although it may depend on exploring Personal Access Token permissions in more details.

TLDR: if I can access private Github feeds with an app by URL (so HTTP GET only), I won't need this feature.

**Comment by hughbris** [comment]
> Why not add an extra tab in Admin in the Pushy section that lists a `git log` response? That might also be useful for the users of Pushy.

It would indeed be useful for users, but (speaking selfishly) not so much for me. Different to the feature I was thinking of, but definitely worth considering as a new feature.

(However, workflow redesign plans, that I haven't documented in public yet, might make it a waste of effort to develop features for this issue and related issues.)
